src/extension.ts

```diff
@@ -28,3 +31,36 @@ export async function deactivate(): Promise<void> {
     return client.stop();
   }
 }
+
+async function activateGemOutdatedButton(context: vscode.ExtensionContext) {
+  const commandId = "updateOutdatedGem";
+
+  gemOutdatedButton = vscode.window.createStatusBarItem(
+    vscode.StatusBarAlignment.Left,
+    0
+  );
+
+  gemOutdatedButton.text = "$(plug) Update Ruby LSP";
```

Comment:
I chose a little plug icon because it looks like this extension's logo! Let me know what you think.

<img width="178" alt="Screen Shot 2022-10-12 at 2 03 01 PM" src="https://user-images.githubusercontent.com/9601737/195415508-7c959563-b2f0-4597-ab74-cb1f597a5fbb.png">